src/components/Link/Link.stories.tsx

```diff
@@ -41,16 +42,22 @@ export default {
 
 const Template: Story<LinkProps> = (args) => (
```

Comment:
Noticing the `Link` stories is not csf v3. Is this one of those story files where there are errors when made into csf v3?

Reply:
Testing it out with a simple story and none of the extras, it seems okay. How about I investigate that in a separate PR?

Reply:
That sounds good to me!